astropy/modeling/core.py

```diff
@@ -3707,9 +3714,11 @@ def outputs_map(self):
                         outputs_map[out] = self.left, self.left.outputs[i]
                 else:  # Get from right
                     if isinstance(self.right, CompoundModel):
-                        outputs_map[out] = r_outputs_map[self.right.outputs[i - len(self.left.outputs)]]
+                        outputs_map[out] = r_outputs_map[self.right.outputs[
+                            i - len(self.left.outputs)]]
```

Comment:
Did you run flake8 with line length 100? I think the old line was less than 100 characters.

Reply:
Its 104 characters unfortuately

Reply:
Oh well. externalities of a consistent style format.